### ISSUE TITLE
Thoughts On A Ships Log

### ISSUE THREAD

**Issue by Miggy4000** [pioneerspacesim/pioneer#204]
This is a feature request.
Was having a think about how a log could be incorporated into the game. Pioneer has a massive galaxy and a ship's log would be useful in managing your travels around it.

Possible features I was thinking about are:

Stores the last 10 systems visited.

Possible to create a list of favourite systems that you frequently visit.

The ability to automatically plot a multiple hyperspace jump taking the most economic route.

Display ETA at the destination system.

Display a warning if you have unpaid fines at the destination system.

Autojump feature fully incorporated into the autopilot, basically one click on the icon and away you go.

Log fully incorporated into the games GUI with a cancel button should you change your mind or want to add or remove waypoints.

Log displays route on the Galactic and Planetary maps.

Log displays icons over systems you have missions in, for example, a triangle for bombing missions, a square for delivery missions and so on. The colour of these icons with be dependant on how soon the missions need to be done. Dark colours for missions that have plenty of time left, brighter colours for those that need to be done soon and flashing icons for those missions that need to be done as soon as possible.

Basically to should be easy to use and serve as a helpful tool for the budding trader or explorer. What does the Pioneer community think?
